### DISCUSSION TITLE
How to use an existing http listner to serve an stager

### DISCUSSION THREAD

**Discussion by dalemazza** [BishopFox/sliver#1379] [Q&A] [answered]
I am currently using a client, then a seperate VM's for the team server and another for a redirector. The redirector is working perfect based on the http-c2.json files.

How can i use the existing http listner to host a stager on the established values in the http-c2.json files.

![image](https://github.com/BishopFox/sliver/assets/66513243/f497feda-d35b-41ac-8ced-d44d7f2d04be)

That is my current set up with the profile pointing to my redirector that then forwards it to my TS. I can see in my logs that the rewrite rules work and proxies the URL on but i get a 404 error back.

URL i have tried http://domain.corp/static/attribute_text_w01_regular.woff

Screenshot of my hht-c2 file
![image](https://github.com/BishopFox/sliver/assets/66513243/82545100-b8f7-4d71-a4a2-a0049f2e7408)

**Comment by rkervella** [comment, marked as answer]
You can't at the moment. As a workaround, you can use the `websites` feature to do that though: generate your shellcode, then use `websites add-content` to host your shellcode file, and then set your HTTP listener to use the created website with `http --website XXX`.
